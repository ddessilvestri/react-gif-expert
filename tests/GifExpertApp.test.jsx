import { render,screen,fireEvent } from "@testing-library/react"
import {GifExpertApp} from '../src/GifExpertApp';


describe('Pruebas en el componente <GifExpertApp />', () => { 
    test('debe de hacer match con el snapshot', () => { 
      
        const newCategory = 'Turtle';

        const {container} =  render(<GifExpertApp/>);
        expect(container).toMatchSnapshot();

     })

})