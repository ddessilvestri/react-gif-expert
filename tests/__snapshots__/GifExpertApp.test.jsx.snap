// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <GifExpertApp /> debe de hacer match con el snapshot 1`] = `
<div>
  <h1>
    GifExpertApp
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Buscar gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch
  </h3>
  <h2>
    Loading...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
